h2o-bindings/bin/gen_R.py

```diff
@@ -329,10 +330,48 @@ def gen_module(schema, algo, module):
     if algo in ["stackedensemble"]:
         yield "  # Error check and build model"
         yield "  model <- .h2o.modelJob('%s', parms, h2oRestApiVersion = %d)" % (algo, 99 if algo in ["svd", "stackedensemble"] else 3)
-        yield "  #Convert metalearner_params back to list if not NULL"
+        yield "  # Convert metalearner_params back to list if not NULL"
         yield "  if (!missing(metalearner_params)) {"
         yield "      model@parameters$metalearner_params <- list(fromJSON(model@parameters$metalearner_params))[[1]] #Need the `[[ ]]` to avoid a nested list"
         yield "  }"
+        yield """
+  model@model$model_summary <- capture.output({
+    print_ln <- function(...) cat(..., sep = "\n")
+    print_ln("Summary of Base Learners:\n")
+    baselearner_summary <- Reduce(
+      function(x, y) {
+        base_learner_summarizer <- function(model) {
+          cbind(
+            list(algorithm = model@algorithm),
+            model@model$model_summary)
+        }
+        rbind(base_learner_summarizer(x), base_learner_summarizer(y))
```

Comment:
What if the summary does not match up in terms of number of rows/columns? Have you tried say a combination of DL, DRF, GBM, and GLM?

Reply:
Oh no I haven't. Good point. Then I have to fill this with NAs.

Reply:
I think we can skip this level of detail on the printout...this will get really messy if we have 100+ models and also @navdeep-G has a good point that we will have to list all possible params for all the algorithms.  

What I had in mind is a table that shows the counts of each algo type:
```
deeplearning          drf          gbm 
           1            1            3 
```
And then we also need to add the `metaleraner_params` list, maybe just print the regular `model_summary` for that algo.